generic-helpers/src/main/java/io/elimu/a2d2/helpers/JedisHelper.java

```diff
@@ -0,0 +1,63 @@
+package io.elimu.a2d2.helpers;
+
+import java.time.Duration;
+
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.clients.jedis.Protocol;
+
+public class JedisHelper {
+
+	static final int JEDIS_PORT = Integer.parseInt(System.getProperty("cache.jedis.port", "6379"));
+	static final String JEDIS_PWD = System.getProperty("cache.jedis.password");
+	static final String JEDIS_HOST = System.getProperty("cache.jedis.host", "localhost");
+	static final Boolean JEDIS_SSL = Boolean.valueOf(System.getProperty("cache.jedis.use_ssl", "false"));
+	static final int JEDIS_TIMEOUT_CONNECTION = Integer.parseInt(System.getProperty("cache.jedis.connection.timeout", "10000")); //milliseconds
+
```

Comment:
Does this need to update readme file with all mentioned variables?

Reply:
@marianbuenosayres Instead of taking all configuration from system property, i think we should build config file and non passowrd/secret configuration we can start taking from that file. obviously not a part of this PR.